### PR TITLE
Refactored Drawer ListTile Widget 

### LINKED ISSUE #136: Refactoring the ListTile Widgets in MainDrawer.dart file into a Custom Widget
There are a Bunch of ListTile Widgets being used in mainDrawer.dart file for navigating across various screens . That patch of Code can be made more clean and maintainable by Creating a custom Widget instead of Hard coding each ListTile Widget . 

If this seems to be a good issue to work upon . I would like to be assigned for this work .

### PR DESCRIPTION
**Pull request by Sahil-kachhap**
Closes #136

### CODE REVIEW

**Comment by Sahil-kachhap** [comment]
It's still not passing sir .

File: lib/UI Components/Drawer_Nav_Item.dart [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+class drawerNavItem extends StatelessWidget {
+
+  final VoidCallback callback;
+  final IconData  iconData;
+  final String navItemTitle;
+
+  drawerNavItem({
```
Comment (Saransh-cpp) [inline lib/UI Components/Drawer_Nav_Item.dart]: Add `const` keyword before the constructor (lint check 3/4)

File: lib/UI Components/Drawer_Nav_Item.dart [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ListTile(
+        onTap: callback,
+        leading:  Icon(iconData),
+        title: Text(
+          navItemTitle,
+          style: TextStyle(fontSize: 18),
```
Comment (Saransh-cpp) [inline lib/UI Components/Drawer_Nav_Item.dart]: Add `const` keyword before TextStyle() (lint check 4/4)

**Review by Saransh-cpp — CHANGES_REQUESTED** [review]
Rename the dart file and use only small letters (something_like_this). (lint issue 1/4)

Use UpperCamelCase for class names (rename the class to DrawerNavItem). (lint issue 2/4)

Final decision will be by mentors:)

**Comment by Sahil-kachhap** [comment]
Thanks @Saransh-cpp . All checks have passed now .

**Pull request merged**